src/singletons/Settings.hpp

```diff
@@ -118,6 +118,8 @@ class Settings : public ABSettings, public ConcurrentSettings
                                           -0.5f};
     // BoolSetting useCustomWindowFrame = {"/appearance/useCustomWindowFrame",
     // false};
+    BoolSetting removeSpacesBetweenEmotes = {
+        "/appearance/removeSpacesBetweenEmotes", false};
```

Comment:
Shouldn't this be in `/emotes/` section (around line 170)?

Reply:
cd7e5ba